Automattic/jetpack
# E2E tests: use wp-env in CI

This PR adds [wp-env](https://www.npmjs.com/package/@wordpress/env) dependency, which relaces both local dockerized and CI E2E environments. 

It would allow us to run E2E tests in the same environment locally and in CI. This should simplify new test development and will make debugging easier. 

Also, I updated `push-allure-artifacts.sh` to cleanup stale built files. 

#### Jetpack product discussion


n/a
#### Does this pull request change what data or activity we track or use?


n/a
#### Testing instructions:






* play with `./tests/e2e/bin/env.sh start/reset` script. It allows you to `start` and `reset` environments.
*  Visit a ngrok instance created by the above command, login with `admin/password`
* Make sure Travis tests are green

#### Proposed changelog entry for your changes:

* n/a
